crm_department/README.rst

```diff
@@ -0,0 +1,76 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+CRM with Department Categorization
+===========
```

Comment:
This might not work with all rst interpretors. Better to put enough `=` to underscore completely the title

You can test it on http://rst.ninjs.org/